doc/i2p.md

```diff
@@ -39,6 +39,12 @@ Core configuration options:
      Ignored if -i2psam is not set. Listening for incoming I2P
      connections is done through the SAM proxy, not by binding to a
      local address and port (default: 1)
+
+-i2ptransientout
+     Generate a transient, one-time I2P address for ourselves when making
+     I2P outbound connection and discard it once the connection is
+     closed. Does not affect the address for accepting incoming I2P
+     connections which is persistent. Ignored if -i2psam is not set.
```

Comment:
In line with the suggestion in `src/init.cpp`
```diff
-     Generate a transient, one-time I2P address for ourselves when making
-     I2P outbound connection and discard it once the connection is
-     closed. Does not affect the address for accepting incoming I2P
-     connections which is persistent. Ignored if -i2psam is not set.
+     Announce a new, transient I2P address to each outbound I2P connection
+     (default: 1). The address for accepting incoming I2P connections
+     is unaffected and persistent. This option is ignored if -i2psam
+     is not set.
```

Reply:
Done (some of it).